hub/constants.lisp

```diff
@@ -2,7 +2,7 @@
 
 (defconst
   MULTIPLIER___STACK_HEIGHT                8      ;; for height increments
-  MULTIPLIER___DOM_SUB_STAMPS             16      ;; for dom/sub stamp business
-  DOM_SUB_STAMP_OFFSET___REVERT            8
-  DOM_SUB_STAMP_OFFSET___SELFDESTRUCT     12
+  MULTIPLIER___DOM_SUB_STAMPS              8      ;; for dom/sub stamp business
+  DOM_SUB_STAMP_OFFSET___REVERT            6
+  DOM_SUB_STAMP_OFFSET___SELFDESTRUCT      7
   )
```

Comment:
I guess these correspond to $\lambda$, $\epsilon_{rev}$ and $\epsilon_{selfdest}$ in the specs. I believe we should at least add a comment make it clear what they are referring to in the specs.

Reply:
Done